tests/cfg.rs

```diff
@@ -0,0 +1,13 @@
+use std::marker::PhantomData;
+
+use derive_where::derive_where;
+
+#[test]
+fn cfg() {
+	#[derive_where(Clone)]
+	struct Test<T> {
+		a: PhantomData<T>,
+		#[cfg(invalid)]
```

Comment:
Seeing this got me thinking. We could also copy all cfg attributes throughout our expansion. But that would increase complexity a lot probably and would make it necessary to adapt every expansion probably.

Reply:
Agreed, I hope this solution works.